### ISSUE TITLE
Switch to a production web server in Rails

### ISSUE THREAD

**Issue by jbull477** [jbull477/tc359_rails_final#2]

**Comment by jbull477** [comment]
@chrisvfritz  I'm not getting this one.  Could you help me through a hangout or something when you have time?

**Comment by chrisvfritz** [comment]
Definitely. I could do a 12pm, 1pm, or 4pm Hangout tomorrow. Let me know if one of those would work.

**Comment by jbull477** [comment]
I will be available at 4pm! Thanks.

**Comment by chrisvfritz** [comment]
Here's [the Hangout link](https://plus.google.com/hangouts/_/g7ttm7hkvnxkzlz2wtlubxn2oea?hl=en) for when we meet btw.

**Comment by jbull477** [comment]
Alright, thanks.

**Comment by chrisvfritz** [comment]
:+1: Looks good.
